scripts/aws/setupdb.sh

```diff
@@ -14,3 +14,14 @@ psql -c "CREATE EXTENSION IF NOT EXISTS postgis;"
 
 # Run migrations
 envdir /etc/mmw.d/env /opt/app/manage.py migrate
+
+# Fetch boundary layer sql files
+FILE_HOST="http://santacruz.internal.azavea.com:9999"
+curl $FILE_HOST/boundary_huc08.sql.tar.gz | tar xz
+curl $FILE_HOST/boundary_huc10.sql.tar.gz | tar xz
+curl $FILE_HOST/boundary_huc12.sql.tar.gz | tar xz
+
+psql -f boundary_huc08.sql
```

Comment:
Yeah, we can separate the two, and the current sql is performing a `COPY` (not `INSERT`s) within the `psql` session, so it shouldn't be too hard to extract that out to stream directly to `COPY` (the geometry is WKB). 

I boosted my services VM numbers before working on this to speed up the data updating, which could be why I didn't have any issues loading.

I'll take some measurements on memory usage and give it a shot to see if it reduces.